### ISSUE TITLE
Extra states not showing in UI

### ISSUE THREAD

**Issue by lapo-luchini** [ebullient/obsidian-task-collector#59]
Hi, newcomer here, I might have misread something, but is it needed to do anything on my part fo enable support/UX for "canceled" and "partial" tasks?
Also, is it possible to append completion date also on normal ticking of a checkbox (instead of using right button completion)? Other plugins do this (but they don't support partial and canceled tasks, so I'd prefer to use this one).

https://user-images.githubusercontent.com/420454/184302060-a4d65aab-7ffa-4fd1-8fe9-c913ed4a0c7b.mp4

**Comment by ebullient** [comment]
https://github.com/ebullient/obsidian-task-collector/blob/main/README.md#settings


> Additional Task Types (renamed in 0.6.5) Specify the set of single characters that indicate in-progress or incomplete tasks.
> 
> default: (space)
> example: > ?! (a space is included along with other values)
> Notes:
> - This is often used with bullet journal (bujo) style tasks, 
> e.g. [>] for deferred items or [/] for items in progress. 
> See [Task states](https://github.com/ebullient/obsidian-task-collector#task-states).
> - The pop-up dialog for marking tasks will create buttons in the order 
 the characers are included in this string.

**Comment by lapo-luchini** [comment]
Yes, I have set `incompleteTaskValues: " >"` already… changing it to `" >/"` changes precious little.

Editing:
<img width="97" alt="image" src="https://user-images.githubusercontent.com/420454/184351164-699a0b07-354c-4eda-ad51-99d48464e7de.png">

Preview:
<img width="97" alt="image" src="https://user-images.githubusercontent.com/420454/184351201-dd210763-9373-4b7b-a315-37c652cb62e4.png">

**Comment by ebullient** [comment]
Ah. You need.. a task snippet. Any task snippet will do. ITS has one, or you can use Snippetor.. Snippetor (another plugin) doesn't need to run all the time. It will help you make your own task snippet so you can define what these should look like.
https://github.com/ebullient/obsidian-snippetor

Literal wall of toggles. But it will create a snippet for you that you can then enable. ;)

Meanwhile, you can map "TC: Mark Task" to a keyboard shortcut (and right click menu) and use that to complete items w/ these alternate values.
